### ISSUE TITLE
mdDialog - IE11/Win8.1 targetEvent no longer causing dialog to open/close from button

### ISSUE THREAD

**Issue by Londovir** [angular/material#3828]
Confirmed on Demo site using #master. Under previous commits, including most of 0.10.1-rc1, using IE11/Win8.1 showed the expected opening of an md-dialog as "growing" from the targetEvent button originator, and "shrinking" down to that area as well.

With recent code change around 0.10.1-rc2, the targetEvent parameter is not getting used by the code to perform this behavior. Instead, IE11/Win8.1 seems to lag after clicking a button, and after about a half second delay the backdrop and dialog appear almost simultaneously on the screen with no/little animation. When dismissing, the dialog dismisses downwards towards the center bottom of the viewport, which seems to be the behavior when no targetEvent is indicated.

I do not see this behavior in Chrome or Firefox, however.

**Comment by Londovir** [comment]
@ThomasBurleson Although this was closed on July 21, I'm still (as of 9/14/15) seeing the behavior I described above failing in IE11. This is running on Windows 10 IE11:

![ie11dialog](https://cloud.githubusercontent.com/assets/11982575/9858637/c620b4b0-5aef-11e5-8c43-a3073ec07587.gif)

IE11 has a number of differences to note:

1) The backdrop does not become semi-transparent, but looks to be completely opaque.
2) The backdrop completes its animation into place, and **then** the dialog animates into place.
3) The dialog does not expand from the clicked button, nor dismiss away to the button either.

Now, for contrast, here's the same in Chrome:

![chromedialog](https://cloud.githubusercontent.com/assets/11982575/9858647/df887f3c-5aef-11e5-8b6c-35f883da8e72.gif)

This is as it is expected. (Performed from most updated IE11 in Windows 10 and Chrome in Windows 10 from MD Demo page.)

If you believe this is an unrelated issue to the original issue here, then let me know and I can create this as a new issue rather than reopen this one, but I don't believe it has been addressed for IE11.

BTW, Edge performs identical to Chrome for me. It is only IE11 that is exhibiting this issue.
